src/components/Editor/index.tsx

```
@@ -1,40 +1,18 @@
-import {
-    useCallback,
-    useContext,
-    useEffect,
-    useRef,
-    useState,
-} from "react";
-
-import {useColorScheme} from "@mui/joy";
+import { useCallback, useContext, useEffect, useRef, useState } from "react";
```

ESLint

[error] Imports must be broken into multiple lines if there are more than 1 elements.

(import-newlines/enforce)